broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -682,11 +682,24 @@ private RemotingCommand deleteSubscriptionGroup(ChannelHandlerContext ctx,
         log.info("deleteSubscriptionGroup called by {}", RemotingHelper.parseChannelRemoteAddr(ctx.channel()));
 
         this.brokerController.getSubscriptionGroupManager().deleteSubscriptionGroupConfig(requestHeader.getGroupName());
-
+        // only cleanConsumerOffsetList maker 
+        this.brokerController.getConsumerOffsetManager().cleanConsumerOffsetList(requestHeader.getGroupName());
         response.setCode(ResponseCode.SUCCESS);
         response.setRemark(null);
         return response;
     }
+    
+ //  private RemotingCommand cleanConsumerOffsetList(ChannelHandlerContext ctx,
```

Comment:
@Liberxue could you remove these useless codes?

Reply:
ok has been cleaned~